services/QuillLMS/client/app/bundles/Evidence/components/activitySlides/postActivitySlide.tsx

```diff
@@ -1,38 +1,85 @@
 import * as React from "react";
 
-export const PostActivitySlide = ({ responses, user, handleClick, }) => {
-
-  function getRevisionCopy() {
-    let count = 0;
-    if(!responses) { return 'You used evidence to write 3 strong sentences!' }
-    Object.keys(responses).map(key => {
-      count += responses[key].length;
-    });
-    if(count <= 3) { return 'You used evidence to write 3 strong sentences!' }
-    // we subtract 3 to account for a student having 3 optimal first attempts
-    const calculatedCount = count - 3;
-    if(calculatedCount === 1) {
-      return 'You made 1 revision!';
-    }
-    return `You made ${calculatedCount} revisions!`;
+import { BECAUSE, BUT, SO } from '../../../Shared/index'
+import { PromptInterface } from "../../../Staff/interfaces/evidenceInterfaces";
+
+const FIRST_STRONG_EXAMPLE = 'first_strong_example';
```

Comment:
We should probably use the defined constants import here as well as in our tests.

Reply:
good catch!